app/controllers/usage_controller.rb

```diff
@@ -188,7 +188,8 @@ def total_users(args:)
   end
 
   def first_plan_date
-    StatCreatedPlan.all.order(:date).limit(1).pluck(:date).first
+    StatCreatedPlan.all.order(:date).limit(1).pluck(:date).first \
+    || Date.today.last_month.end_of_month
```

Comment:
fixes a problem when there are no stats in the db (for example after a fresh seed)